CMakeLists.txt

```diff
@@ -201,7 +205,7 @@ if(VLLM_GPU_LANG STREQUAL "CUDA")
         # Speed up CUTLASS download by retrieving only the specified GIT_TAG instead of the history.
```

Comment:
does this mean we will clone the whole repo after this pr?

Reply:
This was debug cruft, but you're right that would have grabbed the entire git history